### DISCUSSION TITLE
Syntax highlighting and KaTex not working

### DISCUSSION THREAD

**Discussion by ltickett** [McShelby/hugo-theme-relearn#162] [Q&A] [answered]
Sorry, new to hugo and this theme so probably something i'm doing wrong...

Out of the box syntax highlighting and KaTex don't seem to work.

For syntax highlighting, it looks like it is trying to use `markup.highlight.noClasses=false` (despite me not having that in config.toml).

I was able to fix it by generating syntax.css, dropping it into the relevant folder and adding it to the header partial.

I'm not sure how to fix KaTex.

But i'm more trying to understand why these didn't work out of the box, so I can possibly help fix or update documentation to help other going forward.

Thanks

**Comment by ltickett** [comment]
I think I might have found the culprit for syntax highlighting. My default hugo config.tomly contained `pygmentsUseClasses = true`

Commenting that out seems to be using inline styles rather than classes 👍 

Not sure where to look for the math?

**Comment by McShelby** [comment, marked as answer]
Katex is not nativley supported by this theme but it may be supported by one of Hugo's markdown renderer.

**Reply by ltickett** [reply]
Thank you for confirming
